From: Norman Rasmussen <norman@rasmussen.co.za>
Date: Sun, 10 Jan 2010 21:43:44 +0000
Subject: syslog openlog

    Initialize syslog so that we log with our PID as DAEMON.
---
 common/aiccu.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/aiccu.c b/common/aiccu.c
index 9abb334..75aadc8 100755
--- a/common/aiccu.c
+++ b/common/aiccu.c
@@ -58,6 +58,9 @@ bool aiccu_InitConfig()
 	int ret;
 #define CAFILE "ca.pem"
 #endif
+#ifndef _WIN32
+	openlog(NULL, LOG_PID, LOG_DAEMON);
+#endif
 	/* Allocate & Initialize */
 	g_aiccu = (struct AICCU_conf *)malloc(sizeof(*g_aiccu));
 	if (!g_aiccu) return false;
